library/commandline/src/modules/CommandLine.rb

```diff
@@ -1491,45 +1491,13 @@ def UniqueOption(options, unique_options)
       Ops.get_string(cmds, 0)
     end
 
-    def fake_false
-      false
-    end
-
-    def RunFunction(funct)
-      funct = deep_copy(funct)
-      Report.ClearAll
-      my_funct = deep_copy(funct)
-      ret = my_funct.call
-      report = Report.GetMessages(
-        Ops.greater_than(Report.NumWarnings, 0),
-        Ops.greater_than(Report.NumErrors, 0),
-        Ops.greater_than(Report.NumMessages, 0),
-        Ops.greater_than(Report.NumYesNoMessages, 0)
-      )
-      if Ops.greater_than(Builtins.size(report), 0)
-        Yast.import "RichText"
-        Print(RichText.Rich2Plain(report))
-      end
-      ret
-    end
-
-    def RunMapFunction(funct, arg)
-      funct = deep_copy(funct)
-      arg = deep_copy(arg)
-      Report.ClearAll
-      my_funct = deep_copy(funct)
-      ret = my_funct.call(arg)
-      report = Report.GetMessages(
-        Ops.greater_than(Report.NumWarnings, 0),
-        Ops.greater_than(Report.NumErrors, 0),
-        Ops.greater_than(Report.NumMessages, 0),
-        Ops.greater_than(Report.NumYesNoMessages, 0)
-      )
-      if Ops.greater_than(Builtins.size(report), 0)
-        Yast.import "RichText"
-        Print(RichText.Rich2Plain(report))
+    def run_if_exists(commandline, index)
```

Comment:
This entire method is unnecessary :) because wherever it is called the caller has already checked that commandline[index] exists. I guess the `fake_false` logic was meant just to fit into the YCP type system.